spring-pulsar/src/main/java/org/springframework/pulsar/config/AbstractPulsarListenerContainerFactory.java

```diff
@@ -171,11 +175,27 @@ else if (this.autoStartup != null) {
 			instance.setAutoStartup(this.autoStartup);
 		}
 
+		copyProperties(this.containerProperties, instance.getContainerProperties(),
```

Comment:
I like the dynamic nature of this copy method, **but** it is inconsistent w/ the rest of the properties. It makes me wonder what is special about these 3 vs. the other 5 on line 181-185. For consistency I think it would be good to just fold these 3 properties into the `acceptIfNotNull` chain below. Wdyt?

Reply:
ya, that is fine. We can do that.